Navigation/navigation.css

```diff
@@ -0,0 +1,98 @@
+@import url("../../Utility CSS/navbar.css");
+
+.code-container {
+  width: 100%;
+}
+
+.navigation {
+  height: 10rem;
+  margin: 1rem;
+  width: 90%;
+  background-color: #df0c87;
+  border: 1px solid var(--btn-secondary-color);
+  border-radius: 7px;
+  box-shadow: 10px 10px 5px 0px rgba(0, 0, 0, 0.07);
+}
+
+.nav-sitename {
+  font-size: 2rem;
+  width: 20rem;
+  align-self: center;
+}
+
+i {
+  font-size: 3rem;
+}
+
+.navigation,
+.nav__searchbar,
+.nav-other {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+.nav__searchbar {
+  font-size: 3rem;
+  width: 35rem;
+}
+
+.nav-other {
+  font-size: 3rem;
+  width: 35rem;
+}
+
+.icon {
+  position: relative;
+}
+
+.icon-text {
+  font-size: 1.7rem;
+  position: absolute;
+  top: 3rem;
+  left: 0rem;
+}
+
+.anchor {
+  text-decoration: none;
+  color: rgb(219, 204, 204);
+}
+
+.nav-logo {
+  font-weight: bold;
+  font-size: 3rem;
+  color: var(--btn-secondary-color);
+}
+
+input {
+  width: 25rem;
+  height: 3.5rem;
+  border-radius: 5px;
+  outline: none;
+  padding: 4px 8px;
+  border: 2px solid var(--border-color);
+}
+
+i:hover {
+  color: var(--primary-color);
+}
+
+.btn {
```

Comment:
make a generalize button and import button from button.css
and then add styling, if any specific